assets/queries/k8s/anonymous_auth_is_not_set_to_false/query.rego

```diff
@@ -0,0 +1,55 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	container := resource.spec.containers[j]
```

Comment:
Please note that `resource.spec.containers` works with Pods but not with `Deployment`, `DaemonSet`, `Cronjob`, etc. as containers are specified in a different hierarchy level. I'd suggest to use `k8sLib.getSpecInfo` instead.